app/rdf/query-search.ts

```diff
@@ -168,6 +168,10 @@ export const searchCubes = async ({
     ${makeInFilter("theme", themeValues)}
     ${makeInFilter("creator", creatorValues)}
 
+    FILTER(LANG(?name) = LANG(?description))
+    FILTER(LANG(?name) = LANG(?creatorLabel))
+    FILTER(LANG(?name) = LANG(?themeName))
```

Comment:
How does this work?

Reply:
I edited the description, please tell me if it's clearer.